src/renderer/components/watch-video-comments/watch-video-comments.css

```diff
@@ -61,6 +61,43 @@
   margin-top: 0px;
 }
 
+.commentHeartBadge {
+  display: inline-block;
+  position: relative;
+  width: 25px;
+  height: 20px;
+  margin-left: 10px;
+  margin-bottom: -7px;
+}
+
+.commentHeartBadgeImg {
+  position: absolute;
+  left: 0;
+  width: 15px;
+  height: 15px;
+  border-radius: 50%;
+  -webkit-border-radius: 50%;
+}
+
+.commentHeartBadgeWhite {
```

Comment:
Let's remove this and instead give `commentHeartBadgeRed` an outline.

Reply:
How do I give it an outline?
The only way I've done so is to use layers to place a regular white heart icon over the solid red heart icon.
Is there any other way?

Reply:
I was hoping either `outline` or `text-stroke` would have worked just fine, but I wasn't having any luck getting this to work.  I'll consider this solution as fine then.

Reply:
If you are fine with me adding a dependency (@fortawesome/free-regular-svg-icons), I genuinely think it looks better

Reply:
Yeah let's keep the icon we have for now.  If we decide to bring in the other icon pack as well for a future reason then I'd be open to changing this.